dagger-functions/src/main/java/io/odpf/dagger/functions/udfs/python/PythonUdfConfig.java

```diff
@@ -0,0 +1,64 @@
+package io.odpf.dagger.functions.udfs.python;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
+import io.odpf.dagger.common.configuration.Configuration;
+import lombok.Getter;
+
+import static io.odpf.dagger.functions.common.Constants.*;
+
+public class PythonUdfConfig {
+    private static final Gson GSON = new GsonBuilder()
+            .enableComplexMapKeySerialization()
+            .setPrettyPrinting()
+            .create();
+
+    @SerializedName(PYTHON_FILES_KEY)
+    @Getter
+    private String pythonFiles;
+
+    @SerializedName(PYTHON_REQUIREMENTS_KEY)
+    @Getter
+    private String pythonRequirements;
+
+    @SerializedName(PYTHON_ARCHIVES_KEY)
+    @Getter
+    private String pythonArchives;
+
+    @SerializedName(PYTHON_FN_EXECUTION_ARROW_BATCH_SIZE_KEY)
+    private int pythonArrowBatchSize;
+
+    @SerializedName(PYTHON_FN_EXECUTION_BUNDLE_SIZE_KEY)
+    private int pythonBundleSize;
+
+    @SerializedName(PYTHON_FN_EXECUTION_BUNDLE_TIME_KEY)
+    private long pythonBundleTime;
+
+    public int getPythonArrowBatchSize() {
+        if (pythonArrowBatchSize == 0) {
```

Comment:
pythonArrowBatchSize should be a object and not a primitive.

Reply:
why?

Reply:
sorry, miss the last sentence, will update other config to not primitive as well